packages/botonic-react/tests/webchat/storage.test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import TestRenderer, { act } from 'react-test-renderer'
+import { Webchat } from '../../src/webchat/webchat'
+
+describe('TEST: storage ', () => {
+  afterEach(() => {
+    localStorage.clear()
+    sessionStorage.clear()
+  })
+
+  it('Stores botonicState in the localStorage by default', async () => {
+    await act(async () => {
+      TestRenderer.create(<Webchat />)
+    })
+    expect(localStorage.getItem('botonicState')).not.toBeNull()
+    expect(sessionStorage.getItem('botonicState')).toBeNull()
+  })
+
+  it('Stores botonicState in the localStorage', async () => {
+    await act(async () => {
+      TestRenderer.create(<Webchat storage={'localStorage'} />)
```

Comment:
I think you don't need the brackets at {'localStorage'}
try 
`<Webchat storage='localStorage'}`